i18n/README.vi-vn.md

```diff
@@ -0,0 +1,88 @@
+<p align="center">
+<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/web/static/supabase-light-rounded-corner-background.svg"/>
+</p>
+
+---
+
+# Supabase
+
+[Supabase](https://supabase.io) là một giải pháp mã nguồn mở thay thế cho Firebase. Chúng tôi sử dụng công cụ mã nguồn mở cấp doanh nghiệp để xây dựng các chức năng mà Firebase có.
+
+- [x] Cơ sở dữ liệu Postgres tự chủ
+- [x] Cập nhật dữ liệu thời gian thực
+- [x] Xác thực và phân quyền
+- [x] Tự động tạo API
+- [x] Bảng điều khiển
+- [x] Kho lưu trữ
+- [ ] Functions (sắp có)
+
+## Tài liệu
+
+Để có tài liệu đầy đủ, hãy truy cập [supabase.io/docs](https://supabase.io/docs)
+
+## Hỗ trợ & Cộng đồng
+
+- [Diễn đàn cộng đồng](https://github.com/supabase/supabase/discussions). Tốt nhất cho: hỗ trợ xây dựng và thảo luận về các phương pháp hay nhất về cơ sở dữ liệu.
+- [Vấn đề trên GitHub](https://github.com/supabase/supabase/issues). Tốt nhất cho: các lỗi và "bugs" mà bạn gặp phải khi sử dụng Supabase.
+- [Hỗ trợ email](https://supabase.io/docs/support#business-support). Tốt nhất cho: các vấn đề với cơ sở dữ liệu hoặc cơ sở hạ tầng của bạn.
+- [Discord](https://discord.supabase.com). Tốt nhất cho: chia sẻ ứng dụng của bạn hoặc dành thời gian với cộng đồng.
+
+## Trạng thái
+
+- [x] Alpha: Chúng tôi đang thử nghiệm Supabase với một nhóm người dùng kín
+- [x] Public Alpha: Bất kỳ ai cũng có thể đăng ký tại [app.supabase.io](https://app.supabase.io). Nhưng hãy bình tĩnh với chúng tôi, sẽ có một vài lỗi nhỏ.
+- [x] Public Beta: Đủ ổn định cho hầu hết các trường hợp sử dụng không phải trong môi trường doanh nghiệp (production)
+- [ ] Public: Sẵn sàng cho môi trường doanh nghiệp (production)
+
+Chúng tôi hiện đang ở giai đoạn Public Beta. Xem mục "releases" của repo này để nhận thông báo về các bản cập nhật lớn.
+
+<kbd><img src="https://gitcdn.link/repo/supabase/supabase/master/web/static/watch-repo.gif" alt="Xem repo này"/></kbd>
+
+---
+
+## Cách Supabase hoạt động
+
+Supabase là sự kết hợp của các công cụ mã nguồn mở. Chúng tôi đang xây dựng các tính năng của Firebase bằng cách sử dụng các sản phẩm mã nguồn mở, cấp doanh nghiệp. Nếu các công cụ và cộng đồng tồn tại, với MIT, Apache 2 hoặc giấy phép mở tương đương, chúng tôi sẽ sử dụng và hỗ trợ công cụ đó. Nếu công cụ không tồn tại, chúng tôi tự xây dựng và mở mã nguồn của nó. Supabase không phải là phiên bản 1 đổi 1 của Firebase. Mục đích của chúng tôi là cung cấp cho các nhà phát triển trải nghiệm giống như Firebase bằng cách sử dụng các công cụ nguồn mở.
+
+**Kiến trúc hiện tại**
+
+Supabase là một [nền tảng được lưu trữ] (https://app.supabase.io). Bạn có thể đăng ký và bắt đầu sử dụng Supabase mà không cần cài đặt bất kỳ thứ gì. Chúng tôi vẫn đang tạo ra trải nghiệm phát triển "local" - đây hiện là trọng tâm cốt lõi của chúng tôi, cùng với sự ổn định của nền tảng.
```

Comment:
Can you help improve Supabase intro here @phucvo0708

The rest looks ok to me :+1:. Thank you

Reply:
> Supabase là một [nền tảng được lưu trữ]

sounds weird to me

Reply:
Sure, I'll take a quick look at that part.

Reply:
New changes were pushed to address the weird sounded translations that I made. Please take a look at it @phamhieu.